src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -295,6 +295,51 @@ NNVM_REGISTER_OP(reshape_like)
 .add_argument("rhs", "NDArray-or-Symbol", "Second input.");
 
 
+NNVM_REGISTER_OP(split_like)
```

Comment:
Is there no unit test for this?

Reply:
reshape_like doesn't have unit tests either, and this is just a small variation on that op so I didn't have a good model of how to go about adding unit tests. Tell me what to do and I'll do it!

Reply:
Thanks for the initiative! 
You can create a test with some input and compare the output of the operator with the expected output. 
Both forward and backward should be tested. 
https://mxnet.incubator.apache.org/versions/master/how_to/add_op_in_backend.html#unit-test